### PR TITLE
Hm dao4

### PR DESCRIPTION
**Pull request by grom4er**
Added Dao/Service for Car and Driver.
Check all in main

### CODE REVIEW

File: src/main/java/taxiservice/service/CarServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    @Override
+    public Car create(Car car) {
+        return carDao.create(car);
+    }
+
+    @Override
+    public Car get(Long id) {
+        return carDao.get(id).orElseThrow(
+                () -> new RuntimeException(
+                        String.format("Car by %d not found", id)));
```
Comment (vicire) [inline src/main/java/taxiservice/service/CarServiceImpl.java:24]:
```suggestion
        return carDao.get(id).get();
```

File: src/main/java/taxiservice/service/DriverServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    @Inject
+    private DriverDao driverDao;
+
+    @Override
+    public Driver create(Driver driver) {
+        return driverDao.create(driver);
+    }
+
+    @Override
+    public Driver get(Long id) {
```
Comment (vicire) [inline src/main/java/taxiservice/service/DriverServiceImpl.java:20]: Same as in car

**Review by vicire — CHANGES_REQUESTED** [review]
You also need to add yml file for Travis check

**Review by vicire — APPROVED** [review]

File: src/main/java/taxiservice/service/DriverServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        return driverDao.get(id).orElseThrow(
```
Comment (RomanDubovskyi) [inline src/main/java/taxiservice/service/DriverServiceImpl.java:21]: this approach looks good to me, but make the code consistent (get should have similar logic in all models)

**Review by RomanDubovskyi — CHANGES_REQUESTED** [review]
all good, left single comments about code consistency

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**